tests/src/lib.rs

```diff
@@ -21,40 +24,57 @@ mod tests {
         let prior_dot = std::fs::read_to_string(format!("{path}/road_network.dot"))
             .unwrap_or_else(|_| String::new());
 
-        let clip_pts = Some(LonLat::read_geojson_polygon(&format!(
-            "{path}/boundary.json"
-        ))?);
-        let (mut street_network, _) = streets_reader::osm_to_street_network(
-            &std::fs::read_to_string(format!("{path}/input.osm"))?,
-            clip_pts,
-            MapConfig::default(),
-            &mut timer,
-        )?;
+        let clip_pts = if Path::new(format!("{path}/boundary.json").as_str()).exists() {
+            Some(LonLat::read_geojson_polygon(&format!(
+                "{path}/boundary.json"
+            ))?)
+        } else {
+            None
+        };
+
+        let (mut street_network, _) = match Path::new(format!("{path}/input.osm").as_str()).exists() {
```

Comment:
An `if` seems simpler than a match here

Reply:
good catch! switched it to `let (mut street_network, _) = if Path::new(format!("{path}/input.osm").as_str()).exists() ` in the new commits.